airbyte-integrations/connectors/source-stripe/source_stripe/schemas/external_account_cards.json

```diff
@@ -0,0 +1,147 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "object": {
+      "type": [
+        "string",
+        "null"
```

Comment:
Can you please format this one line :

```
type": [ "string", "null"]
```

to be consisten with other schema files

Reply:
Not sure what is happening here. My commit changed this [as is visible in the diff](https://github.com/airbytehq/airbyte/pull/14357/commits/3b0812738dfa2d21b500e091d84d64fd3e4fac5b), but it doesn't seem to be taking effect here.